streams/src/main/java/org/apache/kafka/streams/state/internals/StoreChangeLogger.java

```diff
@@ -56,7 +57,13 @@ void logChange(final K key, final V value) {
         if (collector != null) {
             final Serializer<K> keySerializer = serialization.keySerializer();
             final Serializer<V> valueSerializer = serialization.valueSerializer();
-            collector.send(this.topic, key, value, this.partition, context.timestamp(), keySerializer, valueSerializer);
+            try {
+                collector.send(this.topic, key, value, this.partition, context.timestamp(), keySerializer, valueSerializer);
+            } catch (final StreamsException e) {
```

Comment:
Same as above. The context of the original exception is lost, thus making it harder to debug